expression/builtin.go

```diff
@@ -442,30 +442,33 @@ var funcs = map[string]functionClass{
-	ast.Lcase:           &lowerFunctionClass{baseFunctionClass{ast.Lcase, 1, 1}},
-	ast.Left:            &leftFunctionClass{baseFunctionClass{ast.Left, 2, 2}},
-	ast.Right:           &rightFunctionClass{baseFunctionClass{ast.Right, 2, 2}},
-	ast.Length:          &lengthFunctionClass{baseFunctionClass{ast.Length, 1, 1}},
-	ast.LoadFile:        &loadFileFunctionClass{baseFunctionClass{ast.LoadFile, 1, 1}},
-	ast.Locate:          &locateFunctionClass{baseFunctionClass{ast.Locate, 2, 3}},
-	ast.Lower:           &lowerFunctionClass{baseFunctionClass{ast.Lower, 1, 1}},
-	ast.Lpad:            &lpadFunctionClass{baseFunctionClass{ast.Lpad, 3, 3}},
-	ast.LTrim:           &lTrimFunctionClass{baseFunctionClass{ast.LTrim, 1, 1}},
-	ast.Mid:             &substringFunctionClass{baseFunctionClass{ast.Mid, 3, 3}},
-	ast.MakeSet:         &makeSetFunctionClass{baseFunctionClass{ast.MakeSet, 2, -1}},
-	ast.Oct:             &octFunctionClass{baseFunctionClass{ast.Oct, 1, 1}},
+	ast.ASCII:      &asciiFunctionClass{baseFunctionClass{ast.ASCII, 1, 1}},
+	ast.Bin:        &binFunctionClass{baseFunctionClass{ast.Bin, 1, 1}},
+	ast.Concat:     &concatFunctionClass{baseFunctionClass{ast.Concat, 1, -1}},
+	ast.ConcatWS:   &concatWSFunctionClass{baseFunctionClass{ast.ConcatWS, 2, -1}},
+	ast.Convert:    &convertFunctionClass{baseFunctionClass{ast.Convert, 2, 2}},
+	ast.Elt:        &eltFunctionClass{baseFunctionClass{ast.Elt, 2, -1}},
+	ast.ExportSet:  &exportSetFunctionClass{baseFunctionClass{ast.ExportSet, 3, 5}},
+	ast.Field:      &fieldFunctionClass{baseFunctionClass{ast.Field, 2, -1}},
+	ast.Format:     &formatFunctionClass{baseFunctionClass{ast.Format, 2, 3}},
+	ast.FromBase64: &fromBase64FunctionClass{baseFunctionClass{ast.FromBase64, 1, 1}},
+	ast.InsertFunc: &insertFunctionClass{baseFunctionClass{ast.InsertFunc, 4, 4}},
+	ast.Instr:      &instrFunctionClass{baseFunctionClass{ast.Instr, 2, 2}},
+	ast.Lcase:      &lowerFunctionClass{baseFunctionClass{ast.Lcase, 1, 1}},
+	ast.Left:       &leftFunctionClass{baseFunctionClass{ast.Left, 2, 2}},
+	ast.Right:      &rightFunctionClass{baseFunctionClass{ast.Right, 2, 2}},
+	ast.Length:     &lengthFunctionClass{baseFunctionClass{ast.Length, 1, 1}},
+	ast.LoadFile:   &loadFileFunctionClass{baseFunctionClass{ast.LoadFile, 1, 1}},
+	ast.Locate:     &locateFunctionClass{baseFunctionClass{ast.Locate, 2, 3}},
+	ast.Lower:      &lowerFunctionClass{baseFunctionClass{ast.Lower, 1, 1}},
+	ast.Lpad:       &lpadFunctionClass{baseFunctionClass{ast.Lpad, 3, 3}},
+	ast.LTrim:      &lTrimFunctionClass{baseFunctionClass{ast.LTrim, 1, 1}},
+	ast.Mid:        &substringFunctionClass{baseFunctionClass{ast.Mid, 3, 3}},
+	ast.MakeSet:    &makeSetFunctionClass{baseFunctionClass{ast.MakeSet, 2, -1}},
+	ast.Oct:        &octFunctionClass{baseFunctionClass{ast.Oct, 1, 1}},
+	//This function is a synonym for LENGTH().
+	//See https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_octet-length
```

Comment:
It's better not to comment. We could keep the format of the string functions without it.

Reply:
how about to put it at the end of line?  the go fmt tool did this auto format.

Reply:
You can simply delete the comment.

Reply:
aha.I think both is okay. and I fixed